test/test_special_keywords.py

```diff
@@ -203,7 +203,7 @@ def test_orca_init_hessian():
 
     # Test that the hessian is readable
     new_hess = parse_hessian(s.specific.orca.geom.InHessName)
-    new_hess = np.array(new_hess, dtype=np.float)
```

Comment:
In Python `float`, `np.float64`, np.float` are identical. See this [deprecation paragraph](https://numpy.org/doc/stable/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated)

Reply:
`np.float` (an alias for `builtins.float`) has recently been deprecated though, hence the change.

Reply:
Changing it just plain `float` (without the `np`) would also fix the issue though.